Description: Add exit(0) after fflush(0) to avoid Error 56 on PowerPC
 Fixes Bug#355842: "zh-autoconvert: autogb does not exit correctly on
 ppc machine" reported by Jutta Wrage <jw@witch.westfalen.de> on 2006-03-08.
 .
 Committed to zh-autoconvert (0.3.16-1) on 2006-08-09.
Author: Yu Guanghui <ygh@debian.org>
Origin: vendor
Bug-Debian: https://bugs.debian.org/355842
Forwarded: no
Last-Update: 2015-11-17
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- zh-autoconvert-0.3.16.orig/autogb.c
+++ zh-autoconvert-0.3.16/autogb.c
@@ -142,5 +142,7 @@ int main(int argc,char*argv[])
 		}
 		pbuf=&buff[2];
 	}while((count=read(0,pbuf,MAX_BUFFER))>0);
+
 	fflush(0);
+	exit(0);
 }
